lib/commons/dom/elements-from-point.js

```diff
@@ -29,8 +36,6 @@ dom.elementsFromPoint = function (doc, x, y) {
 
 	// get all elements via elementFromPoint, and remove them from hit-testing in order
 	while ((current = doc.elementFromPoint(x, y)) && elements.indexOf(current) === -1 && current !== null) {
-
-		// push the element and its current style
```

Comment:
Sure! I didn't realize that I had removed it. It has been restarted in https://github.com/dequelabs/axe-core/pull/205/commits/943644b69e3ec172ef81f8857d7b1b5363af385b